foo_amr/foo_input_amr.cpp

```diff
@@ -300,7 +309,7 @@ const unsigned input_amr::m_start = 7;
  * @{
  */
 static input_singletrack_factory_t<input_amr> g_input_amr_factory;
-DECLARE_COMPONENT_VERSION("AMR input","1.1.1","http://angelo.scene.pl/\n(c) 2003-2009, Andrzej Lichnerowicz\nPowered GSM AMR-NB speech codec\n(c) 2001, 3gpp");
+DECLARE_COMPONENT_VERSION("AMR input","1.1.1","http://angelo.scene.pl/\n(c) 2003-2018, Andrzej Lichnerowicz\nPowered GSM AMR-NB speech codec\n(c) 2001, 3gpp");
```

Comment:
Please also change the string to sth along the lines of:
```
2003-2018: Andrzej Lichnerowicz, Quang Nguyễn\nPowered GSM AMR-NB speech codec\n(c) 2001, 3gpp
```
I'd like to get rid of `(c)`, because I am re-licensing all my works as Unlicensed (Public Domain) anyway, and will do that right after your merge. It'll save me from rebuilding the project.

Reply:
Thank you for reviewing. I have applied all your suggestion.